GitHubActions/build.sh

```diff
@@ -35,14 +35,14 @@ MODE="$1"
 
 if [ "$MODE" = "tests" -o "$MODE" = "all" ]; then
   echo "Running MessageKit tests."
-  carthage bootstrap --platform ios
+  carthage checkout && ./GitHubActions/install-dependencies.sh
```

Comment:
Back then, Carthage did not support xcframework.
Nowadays we need to use xcframework to build on both Intel Mac and Apple Silicon like `carthage bootstrap --platform ios --use-xcframeworks`.

However, to use xcframeworks need to vastly change project settings, which leads conflicts in future update.

So I added `./GitHubActions/install-dependencies.sh` script to build frameworks using `xcodebuild` directly.